From 5d90f33f62375f23367d3c5a7badd2d7f1031744 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 24 Jan 2020 11:49:04 +0000
Subject: [PATCH] libalpine: make vecho and qecho return success

vecho should return success even if $VERBOSE is empty. Similar, qecho
should return success even if $QUIET is empty.

fixes !4
---
 libalpine.sh.in | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/libalpine.sh.in b/libalpine.sh.in
index d7b79fa..0f96b9f 100644
--- a/libalpine.sh.in
+++ b/libalpine.sh.in
@@ -26,12 +26,16 @@ echon () {
 
 # echo if in verbose mode
 vecho() {
-	[ -n "$VERBOSE" ] && echo "$@"
+	if [ -n "$VERBOSE" ]; then
+		echo "$@"
+	fi
 }
 
 # echo unless quiet mode
 qecho() {
-	[ -z "$QUIET" ] && echo "$@"
+	if [ -z "$QUIET" ]; then
+		echo "$@"
+	fi
 }
 
 # echo to stderr
-- 
2.25.0

